doc/man3/X509_digest.pod

```diff
@@ -39,21 +40,28 @@ PKCS7_ISSUER_AND_SERIAL_digest
 
 =head1 DESCRIPTION
 
-X509_digest_sig() calculates a digest of the given certificate
-using the same hash algorithm as in its signature with a fallback to B<SHA-256>
-for algorithms where the digest is an integral part of the signature algorithm
-such as with B<EdDSA> signatures.
+X509_digest_sig() calculates a digest of the given certificate I<cert>
+using the same hash algorithm as in its signature, if the digest
+is an integral part of the certificate signature algorithm identifier.
+Otherwise, a fallback hash algorithm is determined as follows:
+SHA512 if the signature alorithm is ED25519,
+SHAKE256 if it is ED448, otherwise SHA256.
+The output parmeters are assigned as follows.
+Unless I<md_used> is NULL, the hash algorithm used is provided
+in I<*default_md> and must be freed by the caller (if it is not NULL).
+Unless I<md_is_fallback> is NULL,
+I<*md_is_fallback> equals 1 if the hash algorithm used is a fallback, else 0.
```

Comment:
`the I<*md_is_fallback> is set to 1 if the hash algorithm used is a fallback, otherwise to 0.`

Reply:
Done